docs/commerce/checkout/checkout_api.md

```diff
@@ -0,0 +1,74 @@
+---
+description: Use PHP API to work with checkouts in Commerce.
+edition: commerce
+---
+
+# Checkout API
+
+To get checkouts and manage them, use the `\Ibexa\Contracts\Checkout\CheckouterviceInterface` interface.
+
+To manipulate checkouts that are stored in sessions, you use `CheckoutService`. 
```

Comment:
That is not correct AFAIK, you should use the interface above for all cases, unless @konradoboza will correct me.

Reply:
@DominikaK is right, exactly like above.